modules/Bio/EnsEMBL/Compara/Utils/Database.pm

```diff
@@ -65,4 +66,39 @@ sub table_exists {
     return scalar( @info );
 }
 
+=head2 db_exists
+
+    Arg[1]      :  Bio::EnsEMBL::Compara::DBSQL::DBAdaptor $compara_dba (mandatory)
```

Comment:
Based on how this is being used here and in `NewPerSpeciesComparaDB`, I'm guessing this line should be changed to something like the following:
```suggestion
    Arg[1]      :  $host (mandatory)
```

Reply:
Thanks, I missed that - good ol' copy-paste